### DISCUSSION TITLE
What does the extension description "better integration with GNOME" mean?

### DISCUSSION THREAD

**Discussion by ondrejkolin** [andyholmes/valent#585] [General]
Better is always ... better. But what's the selling point here? 

Personally I had to open the source code, to learn about what the extension actually does.

Screenshot from https://valent.andyholmes.ca/
![image](https://github.com/andyholmes/valent/assets/2377891/4973abad-7a1b-4733-834f-6fd22c477a97)

**Comment by andyholmes** [comment]
Currently, it makes some modifications to notifications, the screen lock and integrates into the quick settings menu, but nothing much other than that.

Details are sparse because both are still considered alpha-quality software. That means features are frequently added and removed with little notice, and incompatible changes are *guaranteed* to happen that will require users to wipe their configuration and installation.

In other words, this is not a mistake :slightly_smiling_face:

**Reply by ondrejkolin** [reply]
This answer is great. You're right. It really doesn't matter now. Thanks for Valent. :-)
